src/Response/Response.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace DDB\OpenPlatform\Response;
+
+use DDB\OpenPlatform\Exceptions\RequestError;
+use DDB\OpenPlatform\OpenPlatform;
+use RuntimeException;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+/**
+ * @property array $data
+ */
+class Response
+{
+    protected $data;
+
+    /**
+     * @var \Symfony\Contracts\HttpClient\ResponseInterface
+     */
+    protected $response;
+
+    public function __construct(ResponseInterface $response)
+    {
+        $this->response = $response;
+    }
+
+    public function __set(string $name, $value) : void
+    {
+        throw new RuntimeException('Cannot set proporties on responses.');
```

Comment:
I do not think this qualifies as a `RuntimeException` ([Exception thrown if an error which can only be found on runtime occurs.](https://www.php.net/manual/en/class.runtimeexception.php)) but rather a subclass of a LogicException ([This kind of exception should lead directly to a fix in your code. ](https://www.php.net/manual/en/class.logicexception.php))